server/src/test/java/org/elasticsearch/discovery/MasterDisruptionIT.java

```diff
@@ -67,7 +66,6 @@
 /**
  * Tests relating to the loss of the master.
  */
-@ESIntegTestCase.ClusterScope(scope = ESIntegTestCase.Scope.TEST, numDataNodes = 0, transportClientRatio = 0, autoMinMasterNodes = false)
```

Comment:
I think this shouldn't be removed, although the `autoMinMasterNodes = false` can go.